docs/en/operation/Admin-CLI.md

```diff
@@ -64,6 +64,8 @@ The `doctor` command shows Alluxio errors and warnings.
 ```bash
 # shows server-side configuration errors and warnings
 ./bin/alluxio fsadmin doctor configuration
+# shows worker lost storage warnings
```

Comment:
I would reword to '# shows worker storage health errors and warnings'

Reply:
Updated, thanks!